web/celery-entrypoint.sh

```diff
@@ -19,8 +19,8 @@ Package: firefox
 Pin: version 1:1snap1-0ubuntu2
 Pin-Priority: -1
 ' | tee /etc/apt/preferences.d/mozilla-firefox
-apt update
-apt install firefox -y
+apt-get update
+apt-get install firefox -y
```

Comment:
Why adding `-get` actually? That shouldn't be needed anymore.

Reply:
yes my bad, i forget to reset to apt.
I've done some tests to silent apt a little with no luck

Reply:
Fixed